modules/extension/wmts/src/test/java/org/geotools/ows/wmts/model/WMTSCapabilitiesTest.java

```diff
@@ -112,6 +113,19 @@ public void testCreateParser() throws Exception {
         }
     }
 
+    /**
+     * A Layer might have a MatrixSetLink with an identifier that isn't within the MatrixSet's. Such
+     * a link should be removed.
+     */
+    @Test
+    public void testMissingMatrixSetLink() throws Exception {
+        WMTSCapabilities capabilities = createCapabilities("nasa.getcapa.xml");
+        WMTSLayer layer = capabilities.getLayer("Blue_Marble_Extended");
+        TileMatrixSetLink link = layer.getTileMatrixLinks().get("1.5km");
+        Assert.assertTrue(
```

Comment:
PMD complains here:
```
2022-09-28T12:07:07.4503812Z [INFO] --- maven-pmd-plugin:3.14.0:check (default) @ gt-wmts ---
2022-09-28T12:07:07.4514275Z [INFO] PMD version: 6.42.0
2022-09-28T12:07:07.4521543Z [INFO] PMD Failure: org.geotools.ows.wmts.model.WMTSCapabilitiesTest:125 Rule:SimplifiableTestAssertion Priority:3 Assertion may be simplified using assertNull.
2022-09-28T12:07:07.4525551Z [INFO] PMD Failure: org.geotools.ows.wmts.model.WMTSCapabilitiesTest:125 Rule:SimplifiableTestAssertion Priority:3 Assertion may be simplified using assertNull.

```

use `assertNull(..., link)`

Reply:
So that was the problem. Thanks!